README.md

```diff
@@ -30,23 +30,92 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id, and stop the temporary daemon
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-var ipfsd = require('ipfsd-ctl')
+const factory = require('ipfsd-ctl')
+const localController = factory.localController
 
-ipfsd.disposableApi(function (err, ipfs) {
+localController.spawn(function (err, ipfsd) {
+  const ipfs = ipfsd.ctl
+  const node = ipfsd.ctrl
```

Comment:
`s/node/ipfsCtrl`

Essentially, avoid confusing users for having examples that literally rename what the API is giving you. It is like saying "the name it gives is misleading" when in fact it is not.

Reply:
great point! will rename to something more meaningful.